src/Todo.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { useTodo } from "./TodoProvider";
+import IconButton from "@material-ui/core/IconButton";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+export default function Todo({ id, todo, complete }) {
+  const { setStatusTodo, deleteTodo } = useTodo();
+
+  const checkTodo = (e) => setStatusTodo(id, e.target.checked);
+
+  return (
+    <tr>
+      <td>
+        <input type="checkbox" onChange={checkTodo} checked={todo.complete} />
```

Comment:
`todo.complete` does not exist! I was mistaken when advising this change, since as I mentioned I thought the `todo` was referring to the "object" `todo` and not the `name` of the "todo" :)